ui/src/views/Tasks/submitTaskAction.js

```diff
@@ -13,7 +13,18 @@ import ajv from '../../utils/ajv';
 
 export default async ({ task, form, action, apolloClient, taskActions }) => {
   const actions = removeKeys(cloneDeep(taskActions), ['__typename']);
-  const taskGroup = task.taskId === task.taskGroupId ? task : task.decisionTask;
+  // We need to source scopes from somewhere. Typically these come from the task we're given,
```

Comment:
You might want to make your comment < 80 lines wide to appease the linter :)

Reply:
Looks like it is making the build step fail

Reply:
I thought `lint:fix` would that...but I guess not!